extensions/src/main/java/org/pentaho/platform/plugin/services/importer/LocaleImportHandler.java

```diff
@@ -170,7 +176,7 @@ private String extractLocaleCode( RepositoryFileImportBundle localeBundle ) {
     return localeCode;
   }
 
-  private RepositoryFile getLocaleParent( RepositoryFileImportBundle locale, Properties localePropertiesFromIndex ) {
+  protected RepositoryFile getLocaleParent( RepositoryFileImportBundle locale, Properties localePropertiesFromIndex ) {
```

Comment:
Same here, just remove the access modifier and add the attribute.

Reply:
Done